README.md.gotmpl

```diff
@@ -283,14 +283,24 @@ environment variable in the docker-compose file. Then you can debug with the run
 
 ## Development
 
-*CloudControl* supports a decoupled development of features and flavours. If you're missing something, just fork this
-repository, create a subfolder for your new feature under "features" and add these files:
+*CloudControl* supports a decoupled development of features and flavours.
+
+### Features
+
+If you're missing a feature, just fork this repository, copy the feature template from features/.template into a
+new subfolder, check out the comments in the example files, and modify them to your needs.
+
+These files make up a feature:
 
 * `feature.yaml`: A descriptor for your feature with a title, a description and configuration notes
 * `install.sh`: A shell script that is run by CloudControlCenter and should install everything you need
   for your new feature
 * `motd.sh`: (optional) If you want to show some information to the users upon login, put them here.
 
+And an optional, but recommended `.goss` folder for [integration testing](#integration-testing).
```

Comment:
If it's optional, why add it? Especially if no tests are provided?

Reply:
You mean maybe something like this?

```suggestion
And an optional, but recommended [integration test](integration-testing) in a `.goss` folder.
```

Reply:
Ok, I had to read it twice to get your point :)
Yes that's it.
Maybe 'Add optional, but recommended [integration tests](integration-testing) in a `.goss` folder.'